src/main/java/com/assignment1/item/Taxable.java

```diff
@@ -0,0 +1,8 @@
+package com.assignment1.item;
+
+/** Interface to declare methods dealing with Tax calculations. */
+public interface Taxable {
```

Comment:
this interface can also be redesigned. this behavior should be abstracted to the `Item`

Reply:
Given this behavior to the Item entity. Interface is avoided now.